google/cloud/bigtable/README.md

```diff
@@ -42,7 +42,40 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
 
 ## Release Notes
 
-### v0.9.x - TBD
+### v0.10.x - TBD
+
+### v0.9.x - 2019-05
+
+* **Breaking Changes**
+  * Return `google::cloud::future` from `InstanceAdmin` functions.
+  * Remove unused `bigtable::GrpcError`.
+  * Remove Snapshot-related functions, tests, examples, etc.
+* Continue to implement more async APIs (**Note: These are not yet stable**):
+  * `InstanceAdmin`: implement `AsyncSetIamPolicy`, `AsyncDeleteAppProfile`,
+    `AsyncGetAppProfile`, `AsyncCreateAppProfile`, `AsyncGetIamPolicy`,
+    `AsyncTestIamPermissions`, `AsyncUpdateAppProfile`, `AsyncUpdateInstance`,
+    `AsyncCreateCluster`, `AsyncUpdateCluster`, `AsyncCreateInstance`,
+    `AsyncListAppProfiles`.
+  * `TableAdmin`: Implement `AsyncGenerateConsistencyToken`,
+    `AsyncCheckConsistency`, `AsyncListTables`, `AsyncDropAllRows`,
+    `AsyncDropRowsByPrefix`.
+  * `Table`: Implement `AsyncReadModifyWriteRow`.
```

Comment:
Nit: implement as inconsistent capitalization. Personally, to make it more clear/consistent, I would say something like, "For the `Table` class we've added functions ..."